MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressMediaSanitize.h

```diff
@@ -0,0 +1,191 @@
+/** @file
+  Header file for MEDIA_SANITIZE_PROTOCOL interface.
+
+  Copyright (c) Microsoft Corporation.

```

Comment:
Could you add a `<BR>` at the end for doxygen, please?

Reply:
Updated in the latest push. Thanks for reviewing.